modules/authenticating-trustyai-service.adoc

```diff
@@ -13,20 +13,25 @@ endif::[]
 ifdef::cloud-service[]
 * You installed the OpenShift command line interface (`oc`) as described in link:https://docs.openshift.com/dedicated/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the CLI] (OpenShift Dedicated) or link:https://docs.openshift.com/rosa/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the CLI] (Red Hat OpenShift Service on AWS)
 endif::[]
-* Your OpenShift cluster administrator has installed {productname-short}, added you as a user to the OpenShift cluster, and installed the TrustyAI service for the data science project where the models are deployed.
+* Your OpenShift cluster administrator added you as a user to the {openshift-platform} cluster and has installed the TrustyAI service for the data science project where the models are deployed.
```

Comment:
"for the data science project where the models are deployed"

could also be

"for the data science project that contains the deployed models"